### ISSUE TITLE
TypeError: Cannot read property 'options' of undefined in User.get avatarURL

### ISSUE THREAD

**Issue by brussell98** [abalabahaha/eris#217]
Running on Commit: e54e25f

Error stack:
```js
TypeError: Cannot read property 'options' of undefined
  File "/home/brussell/mirai-bot/node_modules/eris/lib/structures/User.js", line 53, col 129, in User.get avatarURL [as avatarURL]
    return this.avatar ? `${CDN_URL}/avatars/${this.id}/${this.avatar}.${this.avatar.startsWith("a_") ? "gif" : this._client.options.defaultImageFormat}?size=${this._client.options.defaultImageSize}` : this.defaultAvatarURL;
  File "/home/brussell/mirai-bot/events/EventNotifications.js", line 124, col 31, in database.findOne.then.settings
    icon_url: member.user.avatarURL
```

**Comment by utf-4096** [comment]
I am currently getting the same issue.

**Comment by macdja38** [comment]
@brussell98 @DracoZ12 are you on latest eris? because this may have been fixed in latest dev eris... also where are you getting the user object you are running .avatarURL from? as in... onMessage event msg.author? or what?

**Comment by SomeoneCares11** [comment]
I am currently getting the same issue.

in `guildMemberAdd` event using `member.avatarURL`

**Comment by macdja38** [comment]
@SomeoneCares11 are you running an up to date copy of the dev branch?

**Comment by SomeoneCares11** [comment]
@macdja38 i'm on the stable version

**Comment by macdja38** [comment]
@SomeoneCares11 can you get me the exact version tag?

**Comment by SomeoneCares11** [comment]
@macdja38 sorry, i just updated my version to the dev branch, i'll try it and let u know

**Comment by macdja38** [comment]
@SomeoneCares11 I tried it out in the latest dev, not seeing any problems with a few test joins, but it could be a rare thing.

**Comment by abalabahaha** [comment]
This should be fixed in 0.7.0. If it doesn't, feel free to comment/reopen.
